ims-react/src/components/SelectExistinUserForm.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
This file contains the search functionality for choosing an existing user to add to a new inquiry.